content/kubermatic/v2.22/installation/local-kkp-installation/_index.en.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Local Kubermatic Kubernetes Platform (KKP) CE Installation"
+linkTitle = "Local Installation"
+date = 2023-06-26T09:49:10+02:00
+weight = 200
+enableToc = true
++++
+
+This page will guide you through using the KKP installer local command `kubermatic-installer local`. This command simplifies and automates the installation of KKP CE using `kind` and a preconfigured KubeVirt seed.
+
+## Pre-Installation Requirements
+
+- **Operating System:** Currently, only Linux is supported.
+- **Disk Space:** A minimum of 10Gi is recommended.
+- **RAM:** At least 8Gi of RAM is needed; 16Gi is recommended for more than a single KubeVirt user cluster.
+- **`kind`** (version 0.17 or higher): Please refer to the [`kind` documentation](https://kind.sigs.k8s.io/docs/user/quick-start/) for installation instructions.
+
+{{% notice warning %}}
+This command is experimental and not intended for production setups.
+{{% /notice %}}
```

Comment:
Please move this notice up to the top, it should be the first thing on the page. I think it would also be helpful to clarify the description, because it almost sounds like once this is no longer experimental, it _will_ be ready for production setups.

Reply:
Sorry to come back and still complain, but I do think the page should mention that the feature is experimental. Maybe it should be two separate warnings at the top of the page though.

Reply:
> Sorry to come back

no worries, sometimes multiple iterations of incremental improvement is the only way to reach a consensus :)

However, two "warnings" on the top looked imho a bit awkward so I kept the most important disclaimer about "no prod" on the top and moved the second disclaimer with "experimental" a bit lower and provided more info about why it's experimental and what to expect from the resource consumption point of view. The second warning is still before installation instructions.